api/src/test/scala/uk/ac/wellcome/platform/api/services/ElasticsearchServiceTest.scala

```diff
@@ -47,40 +47,96 @@ class ElasticsearchServiceTest
   }
 
   it("should return the correct number of results from Elasticsearch") {
+    val works: List[IdentifiedWork] = (0 to 9) { x =>
+      identifiedWorkWith(canonicalId = s"ID-$x", label=s"Work number $x")
+    }
+    val displayWorks = works.map { work: IdentifiedWork =>
+      DisplayWork("Work", work.canonicalId, work.work.label)
+    }
+
+    works.map { insertIntoElasticSearch(_) }
+
+    // Take a slice that starts at the beginning
+    assertSliceIsCorrect(
+      searchService,
+      limit = 4,
+      from = 0,
+      expectedWorks = displayWorks.slice(0, 4)
+    )
+
+    // Take a slice that starts midway through the result set
+    assertSliceIsCorrect(
+      searchService,
+      limit = 4,
+      from = 3,
+      expectedWorks = displayWorks.slice(3, 7)
+    )
+
+    // Take a slice that goes off the end of the results set
+    assertSliceIsCorrect(
+      searchService,
+      limit = 7,
+      from = 5,
+      expectedWorks = displayWorks.slice(5, 10)
+    )
+
+    // Take a slice that is beyond all our results
+    assertSliceIsCorrect(
+      searchService,
+      limit = 10,
+      from = 50,
+      expectedWorks = List()
+    )
```

Comment:
@alicefuzier Hopefully easier to add more tests now. Is this the sort of thing you had in mind?